web-ui/view/organisation.ml

```diff
@@ -0,0 +1,52 @@
+open Git_forge
+
+module Make (M : M_Git_forge) = struct
+  let profile_picture_url org =
+    (* FIXME [benmandrew]: How can we get the GitLab profile pictures? *)
+    match M.prefix with
+    | "github" -> Printf.sprintf "https://github.com/%s.png?size=88" org
```

Comment:
Requesting a png of 88px results in a highly compressed-looking image. We can request a larger image (I chose 200px before) and the rendered size can be set in the HTML, it looks far better.